src/applications/discharge-wizard/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "appName": "Claims Status",
+  "entryFile": "./discharge-wizard-entry.jsx",
+  "entryName": "discharge-upgrade-instructions",
+  "rootUrl": "/discharge-upgrade-instructions",
+  "template": {
+    "layout": "page-react-sidebar.html",
+    "title": "How To Apply For A Discharge Upgrade",
+    "heading": "How to apply for a discharge upgrade",
+    "display_title": "Discharge upgrade",
+    "description": "Want to apply for a military discharge upgrade or correction? Get step-by-step instructions depending on your specific situation. If your discharge is upgraded, you'll be eligible for VA benefits you earned while serving.",
+    "keywords": "upgrade discharge, military discharge upgrade, discharge upgrade",
+    "order": 6,
+    "spoke": "Get records",
+    "collection": "records"
```

Comment:
I think support for these attributes will go away or is already gone due to the sidebar approach from the CMS. I didn't analyze the frontmatter that far though, since that's not really the point of this work.